nncf/tensorflow/strip.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2025 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from nncf.common.utils.backend import copy_model
+from nncf.tensorflow.graph.model_transformer import TFModelTransformer
+from nncf.tensorflow.graph.transformations.commands import TFOperationWithWeights
+from nncf.tensorflow.graph.transformations.commands import TFRemovalCommand
+from nncf.tensorflow.graph.transformations.layout import TFTransformationLayout
+from nncf.tensorflow.graph.utils import collect_wrapped_layers
+from nncf.tensorflow.quantization.quantizers import AsymmetricQuantizer
+from nncf.tensorflow.quantization.quantizers import SymmetricQuantizer
+from nncf.tensorflow.quantization.utils import apply_overflow_fix_to_layer
+from nncf.tensorflow.sparsity.magnitude.operation import BinaryMask
+from nncf.tensorflow.sparsity.rb.operation import RBSparsifyingWeight
+from nncf.tensorflow.sparsity.utils import apply_mask
+
+
+def strip(model: tf.keras.Model, do_copy: bool = True) -> tf.keras.Model:
+    """
+    Implementation of the nncf.strip() function for the TF backend
+    """
+    wrapped_layers = collect_wrapped_layers(model)
```

Comment:
@alexsu52 I just want to highlight. I will move the `nncf.strip(`) call inside the `convert_model()` function, as we discussed. This means that the method will be called for every TensorFlow (TF) model, whether quantized or not. The `collect_wrapped_layers()` function will be called (for any TF model) to check if there are any of our layers. It traverses all layers.